src/pages/billing/Billing.jsx

```diff
@@ -94,28 +95,33 @@ const Billing = () => {
             <div>
                 <Sidenav selectedItem="billing" />
                 <div className="page-content">
-                    {!subscribed && <h3 style={{ marginBottom:"1%", fontSize:"21px"}}>Upgrade <span style={{fontSize:"14px"}}>(currently using free plan)</span></h3>}
+                    {!subscribed && <div>
+                    <h3 style={{ marginBottom:"0", fontSize:"21px"}}>Upgrade cluster</h3>
+                        <p style={{ marginBottom:"1%"}}>This Space Cloud cluster is operating in opensource mode right now. Upgrade the cluster to a paid plan to get increased limits for the cluster</p>
```

Comment:
This text can go inside the select plan component because it is always used in conjunction with the select plan component only